spec/uff_db_loader_spec.rb

```diff
@@ -2,4 +2,14 @@
   it "has a version number" do
     expect(UffDbLoader::VERSION).not_to be nil
   end
+
+  describe "configure" do
+    it "allows to set a container_name dynamically" do
+      UffDbLoader.configure do |config|
+        config.container_name_fn = ->(app_name, environment) { "#{app_name}_#{environment}_db_v15" }
+      end
+
+      expect(UffDbLoader.send(:container_name, "sandbox")).to eq "uff_db_loader_sandbox_db_v15"
```

Comment:
I'd like to use `public_send` in those cases, as I want to prevent accidentally calling a private/protected method. What do you think?

```suggestion
      expect(UffDbLoader.public_send(:container_name, "sandbox")).to eq "uff_db_loader_sandbox_db_v15"
```

Reply:
What about intentionally calling a private method? `container_name` is private. That was the reason I shared my thoughts about the command pattern.

Reply:
Oh, okay, read that wrong then. Nevermind.